resources/definitions/zmorph_i500.def.json

```diff
@@ -0,0 +1,168 @@
+{
+    "version": 2, 
+    "name": "Zmorph i500",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Zmorph",
+        "manufacturer": "Zmorph S.A.",
+        "file_formats": "text/x-gcode",
+        "platform": "zmorph_i500_platform.stl",
+        "platform_offset": [-243, -84, 168],
+        "preferred_quality_type": "fast",
+        "has_variants": true,
+        "variants_name": "Hotend",
+        "preferred_variant_name": "Build hotend 0.4",
+        "preferred_material": "zmorph_pla_silver",
+        "has_materials": true,
+        "machine_has_materials": true,
+        "has_machine_quality": true,
+        "supports_usb_connection": false,
+        "supports_network_connection": false,
+        "exclude_materials": [
+            "3D-Fuel_PLA_PRO_Black",
+            "generic_cffcpe",
+            "generic_cffpa",
+            "generic_gffcpe",
+            "generic_gffpa",
+            "generic_bam",
+            "emotiontech_nylon_1030cf",
+            "emotiontech_acetate",
+            "redd_tpe",
+            "structur3d_dap100silicone",
+            "ultimaker_bam",
+            "volumic_petgcarbone_ultra",
+            "volumic_support_ultra"
+            ],
+        "machine_extruder_trains":
+        {
+            "0": "zmorph_i500_build_module",
+            "1": "zmorph_i500_support_module"
+        }
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "Zmorph i500" },
+        "machine_width": { "default_value": 440 },
```

Comment:
On the product page of this printer, it says 460: https://zmorph3d.com/products/zmorph-i500/

Was this difference intentional?